packages/suite/src/support/messages.ts

```diff
@@ -7392,6 +7387,14 @@ export default defineMessages({
         id: 'TR_DEVICE_CONNECTED_INITIAL_DEVICE_STATE',
         defaultMessage: 'Device connected—complete setup to continue',
     },
+    TR_INITIAL_RECONNECT_DEVICE_TITLE: {
```

Comment:
this is a fix of the state for `device-disconnect-required` that displays complete bs